client/src/intro-exit/Surveys/ExampleExitSurvey.jsx

```diff
@@ -7,9 +7,29 @@ import SurveyWrapper from "../../components/SurveyWrapper";
 import surveyJSON from './ExampleSurvey.json';
 
 export default function ExampleExitSurvey({ next }) {
+    const player = usePlayer();
+
+    if (player.get('tEnd') === null) {
```

Comment:
We'll need to find a way to move this to server side as well, because we can't trust client side clocks to be synchronized (or keep accurate time).